v2/swagger.yaml

```diff
@@ -0,0 +1,629 @@
+          required: false
+          type: string
+        - name: show_transitive
+          in: header
+          description: This is required to enable or disable the transitive support. We need to pass the value as 'true' if transitive is needed. Default value is 'false'.
+          required: false
+          type: string
+        - name: manifest[]
+          in: formData
+          description: A manifest file dependencies.txt for maven, npmlist.json for npm & pylist.json for pypi. Commands to generate these files are mentioned in the README.md
+          required: true
+          type: file
+        - name: filePath[]
+          in: formData
+          description: The absolute or relative path where the manifest file is located. This has to be mandatorily provided with the manifest[] parameter and vice versa.
+          required: true
+          type: string
+
+      responses:
+        '200':
+          schema:
+            $ref: "#/definitions/StackAnalysesPostResponse"
+          description: Response with the unique identifier for the stack analysis request
+        '401':
+          description: Request Unauthorized
+        '400':
+          description: Bad request
+  '/stack-analyses/{request-id}':
+    get:
+      tags:
+        - Scan Services
+      operationId: f8a_scanner.api_v1.stack-analyses-get
+      summary: Get security and other details about an application stack and related recommendations to improve the stack
+      parameters:
+        - in: path
+          name: request-id
+          type: string
+          required: true
+          description: stack analysis request id
+      responses:
+        '200':
+          schema:
+            $ref: "#/definitions/StackAnalysesGetResponse"
+          description: Detailed response for the requested application component
+        '401':
+          description: Request unauthorized
+        '404':
+          description: Data not found
+        '408':
+          description: Request has timed out on server
+definitions:
+  Version:
+    title: Component Version Details
+    description: Component version details
+    properties:
+      pecosystem:
+        type: array
+        items:
+          type: string
+      pname:
+        type: array
+        items:
+          type: string
+      version:
+        type: array
+        items:
+          type: string
+      declared_licenses:
+        type: array
+        items:
+          type: string
+      licenses:
+        type: array
+        items:
+          type: string
+      last_updated:
+        type: array
+        items:
+          type: number
+          format: float
+      vertex_label:
+        type: array
+        items:
+          type: string
+      cve_ids:
+        type: array
+        items:
+          type: string
+      description:
+        type: array
+        items:
+          type: string
+      cm_loc:
```

Comment:
What is the role of this field ?

Reply:
Legacy stuff, carried over from v1. I will fix it.